### PR TITLE
Indexer V2

### PR DESCRIPTION
**Pull request by l-monninger**
# Summary
- **RFCs**: $\emptyset$
- **Categories**:  `protocol-units`, `networks`

1. Updates and integrates Aptos indexer to v2 for gRPC stream. 
2. Uses `localnet` setup to produce consumable TSS with `node_config.indexer_grpc.use_data_service_interface = true;`: https://github.com/aptos-labs/aptos-core/blob/235402ec1683faa8cbd5df098c634d071d2c3000/config/src/config/indexer_grpc_config.rs#L27
3. By default, the stream listens on `0.0.0.0:30734`.

# Changelog

1. Bootstrapping for background tasks needed in gRPC service. 
2. Updates to `docker-compose.yml`. 
4. `run_forever` the gRPC service. 

# Testing

1. e2e tests.
2. Additional transaction stream test included.
3. Manual testing with `grpcui -plaintext localhost:30734. 

# Outstanding issues
1. Indexer processor not yet available.
2. Genesis and other system transactions still seem to be missing `WriteTableItem.data`.

### CODE REVIEW

File: networks/suzuka/suzuka-client/src/tests/indexer_stream.rs [hunk 1]
```diff
@@ -0,0 +1,68 @@
+// pub mod alice_bob;
```
Comment (andyjsbell) [inline networks/suzuka/suzuka-client/src/tests/indexer_stream.rs:1]: Do we need to comment this out or just remove?

**Review by andyjsbell — COMMENTED** [review]
LGTM

**Review by andyjsbell — APPROVED** [review]

File: networks/suzuka/suzuka-client/src/tests/indexer_stream.rs (hunk 1 above)
Comment (l-monninger) [reply networks/suzuka/suzuka-client/src/tests/indexer_stream.rs:1]: It's part of a larger break out. For this PR, I can do whichever.

**Comment by andyjsbell** [comment]
`grpcui 0.0.0.0:30734` gives
> Failed to dial target host "0.0.0.0:30734": dial tcp 0.0.0.0:30734: connect: connection refused

**Pull request merged**